darwin/client.py

```diff
@@ -570,7 +570,7 @@ def parse_version(version_str):
             pass
 
     @property
-    def newer_darwin_version(self):
+    def newer_darwin_version(self) -> Optional[Tuple[int, int, int]]:
```

Comment:
Not sure what you think, but I'd be super happy to introduce our own `typings.py`. In this case
```python
DarwinVersionNumber = Tuple[int, int, int]
```
so here we would do `Optional[DarwinVersionNumber]`

Reply:
When do I think these are necessary? Basically, every time the time is not self-documenting the function signature